tests/Behat/Page/Admin/Invoice/ShowPage.php

```diff
@@ -11,20 +11,6 @@
 
 final class ShowPage extends SymfonyPage implements ShowPageInterface
 {
-    /** @var TableAccessorInterface */
-    private $tableAccessor;
```

Comment:
Why did you decide to get rid of table accessor here?

Reply:
IMO working with `data-test` is easer than `tableAccessor` and after my changes `tableAccessor` was not used